gmail/gmail_dump.py

```diff
@@ -2,9 +2,9 @@
 # Licensed under the MIT License.
 
 import argparse
+import time
```

Comment:
The way I like my imports sorted (by full module name), 'time' follows 'pathlib'.